### ISSUE TITLE
about root

### ISSUE THREAD

**Issue by DajianXu** [Sucareto/Android_HID_Keyboard#8]
Is root permission necessary?Can it be possible by non-root android device.

**Comment by Sucareto** [comment]
Yes, root privileges are required to create simulated HID devices, and this software also needs root access to perform its operations.
